src/components/Scoreboard/Scoreboard.js

```diff
@@ -0,0 +1,40 @@
+import { h, Component } from 'preact';
+import { RoundScore } from "components/RoundScore/RoundScore";
+import { AggregateScore } from "components/AggregateScore/AggregateScore";
+
+const Scoreboard = ({ visibleScore, score }) => {
+    return (
+        <div className={`scores -${visibleScore}`}>
+            <RoundScore score={score} />
+            <AggregateScore score={score} />
```

Comment:
We had a switcher which change current score. Will we have it somewhere outside of `Scoreboard` component?

Reply:
it might be anywhere, it just needs to dispatch a proper action. I can add it to this pr) Shall I?

Reply:
Makes sense, as tab switcher is a part of this component

Reply:
I've added it but without integration with redux.